sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Count.java

```diff
@@ -201,5 +201,11 @@ public boolean equals(Object other) {
     public int hashCode() {
       return getClass().hashCode();
     }
+
+    @Override
+    public String getIncompatibleGlobalWindowErrorMessage() {
+      return "If the input collection uses a windowing strategy other than GlobalWindows, "
```

Comment:
The default message, and the one override I have found, indicate that it has to do with the output PCollection. But upon consideration, you are right and the existing messages are wrong. I don't suppose you'd be willing to follow this PR with one that fixes Top and the default, would you?